code/model/TimedNotice.php

```diff
@@ -178,6 +184,60 @@ public function canCreate($member = null)
     }
 
     /**
+     * Gets any notices relevant to the present time, context and current users
+     *
+     * @return ArrayList
+     **/
+    public static function get_notices()
+    {
+        // analyse the context this message gets requested
+        $context = self::getContext();
```

Comment:
Hello @sheadawson 

I have finally managed to update this. Now  there isn't a get_context() method and the context defaults to CMS. Have a look when you get a chance.

Cheers,
Peter